src/public/js/index.js

```diff
@@ -0,0 +1,79 @@
+
+const OK_RESPONSE_CODE = 200;
+
+document.addEventListener('DOMContentLoaded', init);
+
+function init() {
+	var body = document.getElementsByTagName("body")[0];
+
+	var topDiv = document.createElement("div");
+	topDiv.style = "text-align:center;";
+
+	var helloWorldDiv = document.createElement("div");
+	helloWorldDiv.appendChild(document.createTextNode("Hello world"));
```

Comment:
took it a bit too far, the items that are dom should be in the html element.  This should only contain the js needed